messenger/seqclient.go

```diff
@@ -14,14 +18,15 @@ import (
 
 // SequencerClient is a client for interacting with the sequencer.
 type SequencerClient struct {
-	c        *client.Client
-	signer   *client.Signer
-	nonce    uint32
-	rollupId []byte
+	c              *client.Client
```

Comment:
we probably want to remove the sequencer client if it won't be used here. no sense in keeping both around.